docs/generated/.files_generated

```diff
@@ -1,2 +1,6 @@
+artifacts
```

Comment:
I think you just need to scope your "what was generated" check better. The `$BASETMPDIR` stuff is used by the rest of the CI infra to make sure we can recover temp data after bad tests, etc -- I would 100% rather see us use it than anything else.